ecl/eclccserver/eclccserver.cpp

```diff
@@ -331,6 +331,7 @@ class EclccCompileThread : implements IPooledThread, implements IErrorReporter,
         if (mainDefinition.length())
             eclccCmd.append(" -main ").append(mainDefinition);
         eclccCmd.append(" --timings");
+        eclccCmd.append(" --nostdinc");
```

Comment:
If there ever was a situation where stdinc WAS needed, would they be able to override this?

Reply:
They could by setting --eclcc--nostdinc-1 to 0